<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanesDialog</class>
 <widget class="QWidget" name="PlanesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar">
	<property name="title" stdset="0">
         <string>Planes Configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="Tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Data Source</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" rowspan="3">
        <widget class="QGroupBox" name="BSGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Use BaseStation data</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Host</string>
            </property>
            <property name="buddy">
             <cstring>bsHost</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="bsHost">
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>bsPort</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="bsPort">
            <property name="inputMask">
             <string>00000</string>
            </property>
            <property name="text">
             <string>30003</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="bsConnectButton">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="bsDisconnectButton">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="bsStatus">
            <property name="text">
             <string>Disconnected</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="useBSDB">
            <property name="text">
             <string>Write flights to database specified
to the right</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1" rowspan="3">
        <widget class="QGroupBox" name="databaseGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Use database connection</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="dbHostLabel">
            <property name="text">
             <string>Host</string>
            </property>
            <property name="buddy">
             <cstring>dbHost</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="dbPortLabel">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>dbPort</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="dbUserLabel">
            <property name="text">
             <string>User</string>
            </property>
            <property name="buddy">
             <cstring>dbUser</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="dbPasswordLabel">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>dbPass</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dbHost"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="dbPort">
            <property name="inputMask">
             <string>00000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="dbUser"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="dbPass">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="dbConnectButton">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="dbDriver"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Driver</string>
            </property>
            <property name="buddy">
             <cstring>dbDriver</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="dbDisconnectButton">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="dbStatus">
            <property name="text">
             <string>Disconnected</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="dbName"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="dbNameLabel">
            <property name="text">
             <string>Database</string>
            </property>
            <property name="buddy">
             <cstring>dbName</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="importFileButton">
         <property name="text">
          <string>Import BaseStation Recording</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="connectOnStartup">
         <property name="text">
          <string>Connect automatically on startup</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="reconnectOnConnectionLoss">
         <property name="text">
          <string>Attempt to reconnect on connection loss</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1" rowspan="4">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Trail Style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="col_solid">
            <property name="text">
             <string>Solid Colour</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="col_height">
            <property name="toolTip">
             <string>Height is encoded in the red colour with min height = 0 and max height = 255</string>
            </property>
            <property name="text">
             <string>Colour By Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Customise</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Min height</string>
               </property>
               <property name="buddy">
                <cstring>minHeight</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Max height</string>
               </property>
               <property name="buddy">
                <cstring>maxHeight</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="maxHeight">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
               <property name="value">
                <double>20000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="minHeight">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="col_velocity">
            <property name="toolTip">
             <string>Velocity is encoded in the red channel, vertical rate &lt; 0 is encoded in the green channel and vertical rate &gt; 0 in the blue channel</string>
            </property>
            <property name="text">
             <string>Colour By Velocity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Customise</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Min ground velocity</string>
               </property>
               <property name="buddy">
                <cstring>minVel</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Max ground velocity</string>
               </property>
               <property name="buddy">
                <cstring>maxVel</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Min vertical rate</string>
               </property>
               <property name="buddy">
                <cstring>minVertRate</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Max vertical rate</string>
               </property>
               <property name="buddy">
                <cstring>maxVertRate</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="maxVertRate">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="minVertRate">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="minimum">
                <double>-999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>0.000000000000000</double>
               </property>
               <property name="value">
                <double>-50.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="minVel">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="maxVel">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
               <property name="value">
                <double>500.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="showLabels">
            <property name="toolTip">
             <string>Draw labels with the callsign of the flight next to the icons</string>
            </property>
            <property name="text">
             <string>Show Flight Labels</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useInterp">
            <property name="toolTip">
             <string>Interpolate between datapoints for smoother animation</string>
            </property>
            <property name="text">
             <string>Use Interpolation</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Display Trails For</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="trails_nothing">
            <property name="toolTip">
             <string>Do not show trails at all</string>
            </property>
            <property name="text">
             <string>Nothing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="trails_selected">
            <property name="toolTip">
             <string>Show trails for the currently selected flight</string>
            </property>
            <property name="text">
             <string>Selected Plane Only</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="trails_inView">
            <property name="toolTip">
             <string>Show trails for all the flights currently on the screen</string>
            </property>
            <property name="text">
             <string>Planes Currently In View Only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="trails_all">
            <property name="toolTip">
             <string>Show trails for all flights, regardless of screen position</string>
            </property>
            <property name="text">
             <string>All Planes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Display Colour</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="ColorButton" name="infoTextColorButton">
            <property name="toolTip">
             <string>Color for info text</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="infoTextColorLabel">
            <property name="text">
             <string>Info text color</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextBrowser" name="aboutTextBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
    <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>gui/ColorButton.hpp</header>
  </customwidget>  
 </customwidgets>
 <tabstops>
  <tabstop>Tabs</tabstop>
  <tabstop>BSGroup</tabstop>
  <tabstop>bsHost</tabstop>
  <tabstop>bsPort</tabstop>
  <tabstop>useBSDB</tabstop>
  <tabstop>bsConnectButton</tabstop>
  <tabstop>bsDisconnectButton</tabstop>
  <tabstop>databaseGroup</tabstop>
  <tabstop>dbDriver</tabstop>
  <tabstop>dbHost</tabstop>
  <tabstop>dbPort</tabstop>
  <tabstop>dbUser</tabstop>
  <tabstop>dbPass</tabstop>
  <tabstop>dbName</tabstop>
  <tabstop>dbConnectButton</tabstop>
  <tabstop>dbDisconnectButton</tabstop>
  <tabstop>importFileButton</tabstop>
  <tabstop>showLabels</tabstop>
  <tabstop>useInterp</tabstop>
  <tabstop>trails_nothing</tabstop>
  <tabstop>trails_selected</tabstop>
  <tabstop>trails_inView</tabstop>
  <tabstop>trails_all</tabstop>
  <tabstop>col_solid</tabstop>
  <tabstop>col_height</tabstop>
  <tabstop>minHeight</tabstop>
  <tabstop>maxHeight</tabstop>
  <tabstop>col_velocity</tabstop>
  <tabstop>minVel</tabstop>
  <tabstop>maxVel</tabstop>
  <tabstop>minVertRate</tabstop>
  <tabstop>maxVertRate</tabstop>
  <tabstop>aboutTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
